mvpa2/tests/test_rsa.py

```diff
@@ -73,6 +73,23 @@ def test_PDistConsistency():
     assert_array_almost_equal(res4.samples,cres2)
 
 
+def test_CDist():
+    targets = np.tile(xrange(3),2)
+    chunks = np.repeat(np.array((0,1)),3)
+    ds = dataset_wizard(samples=data, targets=targets, chunks=chunks)
+    # Some distance metrics
+    euc = pdist(data, 'euclidean')
+    pear = pdist(data, 'correlation')
+    city = pdist(data, 'cityblock')
+    cdist_euc = CDist(pairwise_metric='euclidean')
+    cdist_pear = CDist(pairwise_metric='correlation')
+    cdist_city = CDist(pairwise_metric='cityblock')
```

Comment:
why to hardcode into variables, instead of just a loop 
```python
for dist in ('euclidean', 'cor...):
 pd_ = pdist(data, dist)
 cd_= CDist(pariwise_metric=dist)
...
```
?